pybossa/api/__init__.py

```diff
@@ -417,6 +420,80 @@ def task_progress(project_id=None, short_name=None):
     return Response(json.dumps(task_count_dict), mimetype="application/json")
 
 
+@jsonpify
+@blueprint.route('/preferences/<user_name>', methods=['GET'])
+@ratelimit(limit=ratelimits.get('LIMIT'), per=ratelimits.get('PER'))
+def get_user_preferences(user_name=None):
+    """API endpoint for loading account user preferences.
+    Returns a JSON object containing the user account preferences.
+    """
+    if current_user.is_anonymous:
+        return abort(401)
+    if not user_name:
+        return abort(404)
```

Comment:
can be replaced w/ `@login_required` decorator. same can be done for update call

Reply:
Fixed via https://github.com/bloomberg/pybossa/pull/675/commits/f31f16c0ce7b9e1cfdc1bab52f5197da655bc3fb.